aws/resource_aws_iam_role_policy_attachment.go

```diff
@@ -43,7 +45,7 @@ func resourceAwsIamRolePolicyAttachmentCreate(d *schema.ResourceData, meta inter
 		return fmt.Errorf("[WARN] Error attaching policy %s to IAM Role %s: %v", arn, role, err)
 	}
 
-	d.SetId(resource.PrefixedUniqueId(fmt.Sprintf("%s-", role)))
```

Comment:
While its unlikely that operators are currently using the ID of this resource due to its format, we do not currently have a precedent for changing the format of resource IDs and changing them is considered a major breaking change: https://www.terraform.io/docs/extend/best-practices/versioning.html#example-major-number-increments

We can support import without changing this. 👍